python/ray/serve/autoscaling_metrics.py

```diff
@@ -1,17 +1,21 @@
 import threading
 import bisect
 from collections import defaultdict
+import logging
 import time
 from typing import Callable, DefaultDict, Dict, List, Optional
 from dataclasses import dataclass, field
 
 import ray
 
+logger = logging.getLogger(__file__)
```

Comment:
Let's use the Serve logger, see pattern: https://github.com/sihanwang41/ray/blob/219997e449/python/ray/serve/controller.py#L26-L39